workitem/list_type.go

```diff
@@ -96,6 +96,31 @@ func (t ListType) ConvertFromModel(value interface{}) (interface{}, error) {
 	}, t.ComponentType, value)
 }
 
+// ConvertToString implements the FieldType interface
+func (t ListType) ConvertToString(value interface{}) ([]string, error) {
+	if value == nil {
+		return []string{}, nil
+	}
+	valueList, err := ConvertList(func(fieldType FieldType, value interface{}) (interface{}, error) {
+		return fieldType.ConvertToString(value)
+	}, t.ComponentType, value)
+	if err != nil {
+		return nil, errs.Wrapf(err, "Failed to convert list type")
```

Comment:
~~`Failed`~~ -> `failed` and please say what to convert to: `failed to convert list type to string`

Reply:
Fixed aeec122.

Reply:
Same as here https://github.com/fabric8-services/fabric8-wit/pull/2347#discussion_r233423400 : Please rename to `failed to convert list type to string slice`.

Reply:
Fixed 532b987